database-plugins/src/main/java/io/cdap/plugin/db/batch/sink/DBSink.java

```diff
@@ -115,6 +117,10 @@ public void prepareRun(BatchSinkContext context) {
       DBUtils.cleanup(driverClass);
     }
     context.addOutput(Output.of(dbSinkConfig.referenceName, new DBOutputFormatProvider(dbSinkConfig, driverClass)));
+
+    Schema schema = context.getInputSchema();
+    recordLineage(context, dbSinkConfig.referenceName, schema,
+                  schema.getFields().stream().map(Schema.Field::getName).collect(Collectors.toList()));
```

Comment:
schema can be null, your IDE should warn you about possible NPE, please do not ignore them

Reply:
Ack.